.ghci

```diff
@@ -5,4 +5,3 @@
 import TreeSitter.Unmarshal
 import Source.Span
 import Source.Range
-:set prompt "> "
```

Comment:
We should each set this in our per-user `.ghci` file rather than in any project-specific one.